eveonline/managers.py

```diff
@@ -228,6 +228,10 @@ def get_character_by_id(char_id):
         return None
 
     @staticmethod
+    def get_characters_by_api_id(api_id):
```

Comment:
Its just an abstraction layer. Its a little tedious but it means that if the model is refactored in future then it isn't as laborious to change all the calls to the model as there's a single location that is responsible for querying. I can guarantee the behavior to callers and use a proxy model if the changes are sufficiently drastic.

I'm happy to go either way but if it looks like its already being done one way then I try to keep using that pattern.